java/ql/src/experimental/Security/CWE/CWE-036/OpenStream.ql

```diff
@@ -0,0 +1,56 @@
+/**
+ * @name Calling openStream on URLs created from remote source can lead to file disclosure
+ * @description If openStream is called on a java.net.URL, that was created from a remote source
+ *              an attacker can try to pass absolute URLs starting with file:// or jar:// to access
+ *              local resources in addition to remote ones.
+ * @kind path-problem
```

Comment:
@aschackmull - would it be a good idea to tag this query with the CWE number, or isn't that something we do for experimental queries?

Reply:
I don't think that matters too much for now - we can do that in follow-up work.